moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_common.h

```diff
@@ -188,11 +191,10 @@ struct FCLGeometry
   }
 
   std::shared_ptr<fcl::CollisionGeometry> collision_geometry_;
-  boost::shared_ptr<CollisionGeometryData>  collision_geometry_data_;
+  CollisionGeometryDataPtr collision_geometry_data_;
 };
 
-typedef std::shared_ptr<FCLGeometry> FCLGeometryPtr;
-typedef std::shared_ptr<const FCLGeometry> FCLGeometryConstPtr;
+MOVEIT_CLASS_FORWARD(FCLGeometry);
```

Comment:
how about making a temporary ``MOVEIT_CLASS_FORWARD_STD`` in the same macro file? then we use it for the few instanced in the code base that already use STD, and we can then easily unifiy both macros in the future when we switch everything to STD using the original ``MOVEIT_CLASS_FORWARD``?